Dockerfile

```diff
@@ -1,21 +1,62 @@
+# syntax=docker/dockerfile:experimental
+
+# Build stage: Install python dependencies
+# ===
+FROM ubuntu:focal AS python-dependencies
+RUN apt-get update && apt-get install --no-install-recommends --yes python3-pip python3-setuptools
+ADD requirements.txt /tmp/requirements.txt
+RUN --mount=type=cache,target=/root/.cache/pip pip3 install --user --requirement /tmp/requirements.txt
+
+
+# Build stage: Install yarn dependencies
+# ===
+FROM node:12-slim AS yarn-dependencies
```

Comment:
There is an update for `node:13-slim` I have just tried it and it works.

Reply:
cool. I think we might go when we upgrade the version of node that we use in our dev enviroment too.